CHANGELOG/CHANGELOG-v0.5.md

```diff
@@ -0,0 +1,22 @@
+# Changelog since v0.4.0
+All notable changes to this project will be documented in this file.
+
```

Comment:
thanks, would you also add the contributor to each item?

Reply:
done.